directory_sso_api_client/user.py

```diff
@@ -73,9 +73,11 @@ def get_last_login(self, start=None, end=None, authenticator=None):
             authenticator=authenticator,
         )
 
-    def create_user(self, email, password, authenticator=None):
+    def create_user(self, email, password, authenticator=None, **kwargs):
         url = self.endpoints['user_create']
```

Comment:
Suggest just add phone number as an optional param rather than use kwargs. It looks cleaner.

Reply:
Done